### PR TITLE
Use "match case" instead, add new servers

### PR DESCRIPTION
**Pull request by Rignchen**
while looking in the code I realise there was a lot of
```py
if variable == "value 1":
    ...
elif variable == "value 2":
    ...
elif variable == "value 3":
    ...
elif ...
```
wich can easily be change to a switch case:
```py
match variable:
    case "value 1":
        ...
    case "value 2":
        ...
    case "value 3":
        ...
    case ...
```
wich is both a cleaner and more optimized way to do the same thing

### CODE REVIEW

**Comment by HoodieRocks** [comment]
Match case was only recently added in 3.10, I don't know what version our host is using though.

**Review by FIyrr — COMMENTED** [review]
what exactly does this do? :P (reffering to the convention.debug thingy)

File: cogs/commands/invite.py [hunk 1]
```diff
@@ -85,6 +86,12 @@ async def invite(
                     title=("**🚪Minecraft Invite**"),
                     description="Join the official minecraft discord server using this link: https://discord.gg/minecraft",
                 )
+            case "dataworld (fr)":
+                embed = disnake.Embed(
+                    color=disnake.Colour.orange(),
+                    title=("**🚪Data World Invite**"),
+                    description="Rejoins Data World pour obtenir de l'aide sur les datapacks avec le lien: https://discord.gg/5y5FBz5",
```
Comment (FIyrr) [inline cogs/commands/invite.py:93]: Since the bot is english, could you maybe translate this to english too?

**Review by FIyrr — CHANGES_REQUESTED** [review]
Since the bot is english, i think it would be more fitting to have the description in english here too :P

**Comment by Rignchen** [comment]
> what exactly does this do? :P (reffering to the convention.debug thingy)

the convention.debug is a tag you manualy give yourself when you dev cause you don't want people who download your datapack to see debug messages (an exemple of message we don't want them to see is the tellraw in the load.mcfunction cause if every datapack send one then the chat just gets spam)

**Comment by Rignchen** [comment]
> Since the bot is english, i think it would be more fitting to have the description in english here too :P

I chosed to translate the description of the server in french as the server is in french (that's why there's a "(fr)" in the name) so I thaught that people who would request this link would prefer a french description

**Comment by Rignchen** [comment]
> Match case was only recently added in 3.10, I don't know what version our host is using though.

python 3.10 was released on Oct. 4, 2021, if your host is still on an older version you should maybe consider looking for a new one

**Comment by HoodieRocks** [comment]
Valid, LGTM, but flyne gets the last decision obv

**Comment by FIyrr** [comment]
> > Since the bot is english, i think it would be more fitting to have the description in english here too :P
> 
> I chosed to translate the description of the server in french as the server is in french (that's why there's a "(fr)" in the name) so I thaught that people who would request this link would prefer a french description

i know that the server is in french and understand your argument, but at the same time someone else might just be browsing through servers included in the command and be confused by a french description

**Comment by FIyrr** [comment]
> > what exactly does this do? :P (reffering to the convention.debug thingy)
> 
> the convention.debug is a tag you manualy give yourself when you dev cause you don't want people who download your datapack to see debug messages (an exemple of message we don't want them to see is the tellraw in the load.mcfunction cause if every datapack send one then the chat just gets spam)

the tellraw in the load function is there to let people know that the pack sucessfully loaded?
this is not just for the devs convenience but especially for everyone using the pack as that message might also contain information about sources and contributors to the pack
we better don't start randomly including convention related stuff in the bot since this is meant to be a completely "vanilla" datapack template and we don't want to force people to use conventions

**Comment by FIyrr** [comment]
> > Match case was only recently added in 3.10, I don't know what version our host is using though.
> 
> python 3.10 was released on Oct. 4, 2021, if your host is still on an older version you should maybe consider looking for a new one

yea the vps is running python 3.10.4 so that won't be a problem

**Pull request merged**